Composer/packages/client/src/components/Header.tsx

```diff
@@ -155,6 +161,18 @@ export const Header = () => {
   const webchatEssentials = useRecoilValue(webChatEssentialsSelector);
   const { setWebChatPanelVisibility } = useRecoilValue(dispatcherState);
   const [hideBotController, hideBotStartController] = useState(true);
+  const [showGetStarted, setShowGetStarted] = useState<boolean>(false);
+  const [showTeachingBubble, setShowTeachingBubble] = useState<boolean>(false);
+
+  // These are needed to determine if the bot needs LUIS or QNA
+  // this data is passed into the GetStarted widget
+  // ... if the get started widget moves, this code should too!
+  const dialogs = useRecoilValue(dialogsWithLuProviderSelectorFamily(projectId));
+  const luFiles = useRecoilValue(luFilesState(projectId));
+  const qnaFiles = useRecoilValue(qnaFilesState(projectId));
+  const requiresLUIS = BotIndexer.shouldUseLuis(dialogs, luFiles);
```

Comment:
So Im guessing here the indexer checks if any bot in the project needs luis or qna? So a multi bot template would be indexed on accordingly?

Reply:
Yes! It will inspect the child projects and return the combined status.  Setting the root bot value applies it to all the children by default.